dnsserv.go

```diff
@@ -1,24 +1,45 @@
 package main
 
 import (
-	"github.com/twtiger/toy-dns-nameserver/reqhandler"
+	"fmt"
+	"log"
 	"net"
 )
 
+const dnsMsgLength = 512
+
 func getUDPAddr() *net.UDPAddr {
 	return &net.UDPAddr{
 		IP:   net.ParseIP("127.0.0.1"),
-		Port: 53,
+		Port: 8888,
```

Comment:
8888 is a commonly used port for applications that are run locally. How about using 8853?